### ISSUE TITLE
Restrict search space and broaden search

### ISSUE THREAD

**Issue by ToDo125** [automl/AutoNNUnet#3]
### Limit Search Space
Revisit the default configs and limit the search space.
Some hyperparameters are known to have limited impact on the result, or already have a clear best choice (e.g. loss function).

### Broaden Search
At the same time, a lot of budget is currently spent on deep (long) runs, even though early performance already seems to be a good proxy for final nnU-Net performance. Allowing more configs to be sampled, while limiting how many get promoted per rung and how long each is trained, could improve overall search efficiency.
